test/rules/indentation/indentation.js

```diff
@@ -0,0 +1,70 @@
+var runTest = require('../rule-test-base')(require('../../../src/rules/indentation.js'),
```

Comment:
Do you mind breaking this into multiple definitions/lines? It took me a min or 2 to realize what we're doing.

Reply:
Yeah no worries, have split it up a bit.